pyomo/contrib/mindtpy/mip_solve.py

```diff
@@ -17,18 +17,17 @@
 from pyomo.core import Constraint, minimize, value
 from pyomo.core.expr import current as EXPR
 from math import fabs
-
 from pyomo.repn import generate_standard_repn
-
 from pyomo.common.dependencies import attempt_import
+from pyomo.contrib.mindtpy.util import generate_Norm1_objective_function
```

Comment:
Let's avoid function names with a combination of camelCase and this_naming_style.